### PR TITLE
Review packaging bundling of gems js requirements recommends

### PR DESCRIPTION
**Pull request by jordimassaguerpla**
Review of how we package (RPM) portus.

Add gems as sources
Autogenerate autoprovides for js files
Remove mariadb  requirement
Add kubectl requirement

### CODE REVIEW

**Comment by mssola** [comment]
> Add kubectl requirement

What do you mean by this? I see nothing on this regard (as it should) :confused:

**Comment by jordimassaguerpla** [comment]
@mssola : https://github.com/SUSE/Portus/pull/1948/commits/88f2395330917400566fae9af74af29db2227f4a

**Comment by mssola** [comment]
@jordimassaguerpla so you meant `portusctl`, not `kubgectl` :wink:

File: packaging/suse/portus.spec.in [hunk 1]
```diff
@@ -18,6 +18,15 @@
 %define branch    __BRANCH__
 %define portusdir /srv/Portus
 
+%define fix_sheb() ( \
```
[the remark this answers is not in the document]
Comment (jordimassaguerpla) [reply packaging/suse/portus.spec.in]: Why not? Is there a reason not to use a macro?

[the remark this answers is not in the document]
Comment (Vogtinator) [reply packaging/suse/portus.spec.in]: It's easier to read (no \\ needed) and does not change behaviour depending on context.

**Comment by jordimassaguerpla** [comment]
@MaximilianMeister : good catch. I added a commit defining fix_sheb after

**Comment by mssola** [comment]
Thanks @jordimassaguerpla! :clap:

**Pull request merged**